app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
Kindly use `author` for the association then reference the `User` class. 

`belongs_to :author, class_name: 'User'`